src/test/java/core/basesyntax/model/FruitTransactionTest.java

```diff
@@ -0,0 +1,74 @@
+package core.basesyntax.model;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class FruitTransactionTest {
+
+    @Test
+    public void findOperatorBalanceByLetter_ok() {
```

Comment:
```suggestion
    public void findOperationByLetter_Balance_ok() {
```

same naming for other tests 
There are a lot of ways to name your test methods. The main point is that they should have informative names and be consistent along with other developers in your team. For this task use such convention: `<methodUnderTest>_<state>_<expectedBehavior>`; 
For example, if we are testing the method register with a null user's age the test method name should be `register_nullAge_notOk`. notOk is because the test expects the register method to throw an exception.

Reply:
fixed